src/util/strb.h

```diff
@@ -159,7 +159,21 @@ static inline void strb_appendb(strb *sb, strb *sb2) {
  *
  * A format error will place the strb in error mode.
  */
-GPUARRAY_LOCAL void strb_appendf(strb *, const char *f, ...);
+void strb_appendf(strb *, const char *f, ...);
+
+/*
+ * Reads from the file specified by the given file descriptor.
+ *
+ * A read error will place the strb in error mode.
+ */
+void strb_read(strb *, int fd, size_t sz);
```

Comment:
I think there is an error in this line that prevent compilation... why only "strb *," and no name for that parameter? Why does jenkins passed?